guide/project_examples.md

```diff
@@ -1560,6 +1561,45 @@ It is still possible to experiment with xmake to build std modules in lower vers
 
 For a discussion see: [#3255](https://github.com/xmake-io/xmake/pull/3255)
 
+##### Hello, Modules – A Step-by-Step Walkthrough
+
+1. Install the llvm toolchain, compiling manually is recommended, make sure you have libcxx installed
+
+2. Patch your llvm toolchain,by cloning this [repo](https://github.com/mccakit/xmake_llvm_patch) and putting its contents to C:\llvm\share\libc++\v1\std
```

Comment:
Here too, there is too much Windows-specific stuff, and a lot of it is based on third-party repositories.